swagger.yaml

```diff
@@ -1093,6 +1094,264 @@ paths:
               schema:
                 $ref: "#/components/schemas/Error"
 
+  "/card-comments":
+    post:
+      operationId: createComment
+      summary: Add a comment to a Trello card
+      description: Creates a new comment on a specified Trello card.
+      tags:
+        - Trello
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                cardId:
+                  type: string
+                  description: The ID of the Trello card.
+                comment:
+                  type: string
+                  description: The comment text.
+      responses:
+        "200":
+          description: Comment created successfully.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  cardId:
+                    type: string
+                    description: The ID of the Trello card.
+                  comment:
+                    type: string
+                    description: The comment text.
+        "400":
+          description: Missing required parameters.
+        "500":
+          description: Failed to create comment.
+
+  "/card":
+    post:
```

Comment:
Throughout the spec we typically order the get (list) and then post (create) for a given endpoint. Can you make sure where relevant you order your new endpoints in the same way- it just makes it easier to check the spec as it grows.